server/views/pages/functionalSkills/macros/sortableTable.njk

```diff
@@ -0,0 +1,29 @@
+{#  -------------------------------------------------------------
+    Sortable table for the functional skills page
+    `config` argument is an object with the following properties:
+      * id - the id of the sortable table
+      * assessmentType - the name of the data array to loop through { englishSkills, mathsSkills, digitalSkills }
+    ------------------------------------------------------------- #}
+{% macro sortableTable(config) %}
+    <table class="govuk-table" data-module="moj-sortable-table" data-qa="{{ config.id }}-sortable-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Name</th>
+          <th scope="col" class="govuk-table__header"{% if config.assessmentType.length > 1 %} aria-sort="none"{% endif %}>Location</th>
+          <th scope="col" class="govuk-table__header"{% if config.assessmentType.length > 1 %} aria-sort="desc"{% endif %}>Assessed on</th>
```

Comment:
This wasn't included in the ticket, so we can undo it if you like; however there didn't feel much point in being able to sort a table by the headers, if there was only 1 assessment!

Reply:
Yeah, understood;
I think it would be worth checking that with Ben and/or Karen though 👍